imqsauth/http.go

```diff
@@ -619,6 +628,63 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 		central.Central.Log.Warnf("Group '%v' not found: %v", groupname, e)
 		authaus.HttpSendTxt(w, http.StatusNotFound, fmt.Sprintf("Group '%v' not found: %v", groupname, e))
 	}
+
+	// get all identities and their groups
```

Comment:
The error paths earlier in this function should return after sending the http response, instead of continuing to broadcast a false change notification.

I think the broadcast portion of this function should probably be a function on it's own - simply to stick to the "function must fit on the screen" rule of thumb, and it seems plausible that one might want to further chop up the broadcast function into smaller functions in future.

Do not write `\n` in log messages.

In an error such as 'Error reading identities', rather give more context, otherwise one can't deduce where the error originated. For example: `Unable to broadcast change, unable to read identities: %v`

Why initialize the `modifiedIdentities` slice capacity to 10? That's a very arbitrary number. Rather just leave it blank. Also, rather use this syntax to initialize empty slices and maps:
```go
modifiedIdentities := []string{}
```
You can read through the Go standard library to get a feeling for idiomatic Go code.